rectangles.js

```diff
@@ -0,0 +1,26 @@
+
+function area(rectangle){
+  return rectangle["height"] * rectangle["width"]
+}
+
+function perimeter(rectangle){
+  return (rectangle["height"]*2) + (rectangle["width"]*2)
+}
+
+function diagonal(rectangle){
+  return Math.sqrt((Math.pow(rectangle["height"], 2) + Math.pow(rectangle["width"], 2))).toFixed(2)
```

Comment:
You can also call `rectangle.height`. Kind of a nice feature of JS objects compared to Ruby hashes.